File: app/dbmodels/Netpoint.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'Netpoint',
    'menu'          => ''
);
$table['fields'] = array(
    'id'           => DBModels::type('increments',
                                     array('visible' => false)),
    'location_id'  => DBModels::type('integer'),
    'name'         => DBModels::type('string'),
    'comment'      => DBModels::type('text'),
);


$table['relationships'] = array(
    'location' => array(
        'type' => 'belongsTo',
        'item' => 'Location'),
);
include "_commonentitylink.php";

File: fusionglpi/database/netpoint.py
from fusionglpi import db


class Netpoint(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    entity_id = db.Column(db.Integer, db.ForeignKey('entity.id'))
    location_id = db.Column(db.Integer, db.ForeignKey('location.id'))
    name = db.Column(db.String(255, None, True))
    comment = db.Column(db.Text)

    networkport_ethernets = db.relationship('NetworkportEthernet',
                                            backref='netpoint')
